### PR TITLE
add shortcut header include/bredis.hpp

### PR DESCRIPTION
**Pull request by omegacoleman**
#26 
following example of https://github.com/boostorg/asio/blob/develop/include/boost/asio.hpp

### CODE REVIEW

**Comment by basiliscos** [comment]
Looks good!

Could you, please, add a few more things:
- copyright & license header 
- briefly mention the `bredis.hpp` in the documentation (`readme.md`) to let it be usable for others
- add yourself into contributors  (tail of `readme.md`)

**Comment by omegacoleman** [comment]
> 
> 
> Looks good!
> 
> Could you, please, add a few more things:
> 
>     * copyright & license header
> 
>     * briefly mention the `bredis.hpp` in the documentation (`readme.md`) to let it be usable for others
> 
>     * add yourself into contributors  (tail of `readme.md`)

all done :)

File: include/bredis.hpp [hunk 1]
```diff
@@ -0,0 +1,19 @@
+//
+//
+// Copyright (c) 2017 Ivan Baidakou (basiliscos) (the dot dmol at gmail dot com)
```
Comment (basiliscos) [inline include/bredis.hpp]: It's 2019 now, as far as I know :) And you are `Stephen Coleman`, right? :)

**Comment by basiliscos** [comment]
> all done :) 

one more thing, please: add it to same example files (or to some test-files) to make sure it is covered by CI.

File: include/bredis.hpp (hunk 1 above)
Comment (omegacoleman) [reply include/bredis.hpp]: lol right, I just copied from another random file. shall I change it?

Comment (basiliscos) [reply include/bredis.hpp]: yep, sure u are the author :)

**Comment by omegacoleman** [comment]
> 
> 
> > all done :)
> 
> one more thing, please: add it to same example files (or to some test-files) to make sure it is covered by CI.

create one or choose one?

**Comment by basiliscos** [comment]
>   create one or choose one?

just pick any one you like

**Comment by omegacoleman** [comment]
> 
> 
> > create one or choose one?
> 
> just pick any one you like

done, and the heading thing is amended

**Comment by basiliscos** [comment]
Perfect! Thank you for your contribution!

**Pull request merged**